test/compress/collapse_vars.js

```diff
@@ -2451,3 +2451,76 @@ issue_2313_2: {
     }
     expect_stdout: "0"
 }
+
+issue_2319_1: {
+    options = {
+        collapse_vars: true,
+        unused: true,
+    }
+    input: {
+        console.log(function(a) {
+            return a;
+        }(!function() {
+            return this;
+        }()));
+
```

Comment:
whitespace

Reply:
Thanks for catching it - too late in the day for me :sweat: